charts/immich/values.yaml

```diff
@@ -14,12 +14,9 @@ env:
   TYPESENSE_ENABLED: "{{ .Values.typesense.enabled }}"
   TYPESENSE_API_KEY: "{{ .Values.typesense.env.TYPESENSE_API_KEY }}"
   TYPESENSE_HOST: '{{ printf "%s-typesense" .Release.Name }}'
-  IMMICH_WEB_URL: '{{ printf "http://%s-web:3000" .Release.Name }}'
-  IMMICH_SERVER_URL: '{{ printf "http://%s-server:3001" .Release.Name }}'
-  IMMICH_MACHINE_LEARNING_URL: '{{ printf "http://%s-machine-learning:3003" .Release.Name }}'
```

Comment:
This one is still in use I believe, even though you can set it from the gui, but I am also fine removing it

Reply:
Good catch, thanks! It's no longer documented, should we put it back into the docs pages then?

Reply:
It is a weird one. It is part of the config and so I just want to remove it, but it is still here for backwards compatibility with the AIO. I think it is fine to leave undocumented for now.